teps/0114-custom-tasks-beta.md

```diff
@@ -203,6 +203,15 @@ own specification.
 
 ##### Feature Gates
 
+##### New Feature Flag `custom-task-version`
+
+Support both `v1alpha1.Run` and `v1beta1.CustomRun` for 6 months after Beta release so that custom
```

Comment:
Do we want to document when the default will change ? (from `v1alpha1.Run` to `v1beta1.CustomRun`) After the 6 months, in the middle, … ?

Reply:
Thanks Vincent~ ❤️ Rephrased the doc, and added a timeline of switching versions~